v1/accounts/accounts.gunk

```diff
@@ -248,12 +248,65 @@ type UpdateAccountRequest struct {
 	Description string `pb:"2" json:"description"`
 }
 
+// UpdateAccountStatusRequest is the request envelope to update
+// the account status.
+type UpdateAccountStatusRequest struct{
+	// AccountID is the account unique identifier.
+	AccountID string `pb:"1" json:"account_id"`
+
+	// Status it the status to be updated
+	// TODO: Change the string parameters into type
+	// once the status of the account is identified.
```

Comment:
this TODO will show up as part of the docs; move it to a separate comment.

Reply:
I will move this to implementation side. I'll remove the TODO for now